examples/react/i18n-paraglide/README.md

```
@@ -0,0 +1,141 @@
+```
+
+2. Add the vite plugin to your `vite.config.ts`:
+
+```diff
+import { defineConfig } from 'vite'
+import react from '@vitejs/plugin-react'
+import { tanstackRouter } from '@tanstack/router-plugin/vite'
++import { paraglideVitePlugin } from "@inlang/paraglide-js";
+
+export default defineConfig({
+	plugins: [
```

markdownlint-cli2 (0.18.1)

Hard tabs
Column: 1

(MD010, no-hard-tabs)